cmd/alias-list.go

```diff
@@ -43,110 +45,122 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 EXAMPLES:
-  1. List all hosts.
+  1. List all aliases.
      {{.Prompt}} {{.HelpName}}
 
-  2. List a specific host.
+  2. List a specific alias.
      {{.Prompt}} {{.HelpName}} s3
 `,
 }
 
 // Input argument validator..
-func checkConfigHostListSyntax(ctx *cli.Context) {
+func checkAliasListSyntax(ctx *cli.Context) {
 	args := ctx.Args()
 
 	if len(ctx.Args()) > 1 {
 		fatalIf(errInvalidArgument().Trace(args...),
-			"Incorrect number of arguments to list hosts.")
+			"Incorrect number of arguments to alias list command.")
 	}
 }
 
-func mainConfigHostList(ctx *cli.Context) error {
-	checkConfigHostListSyntax(ctx)
+func mainAliasList(ctx *cli.Context, deprecated bool) error {
+	checkAliasListSyntax(ctx)
 
-	// Additional command speific theme customization.
+	// Additional command specific theme customization.
 	console.SetColor("Alias", color.New(color.FgCyan, color.Bold))
 	console.SetColor("URL", color.New(color.FgYellow))
 	console.SetColor("AccessKey", color.New(color.FgCyan))
 	console.SetColor("SecretKey", color.New(color.FgCyan))
 	console.SetColor("API", color.New(color.FgBlue))
-	console.SetColor("Lookup", color.New(color.FgCyan))
+	console.SetColor("Path", color.New(color.FgCyan))
```

Comment:
shouldn't we conditionally set "Path" or "Lookup" based on `deprecated` flag?

Reply:
> shouldn't we conditionally set "Path" or "Lookup" based on `deprecated` flag?

Correct, but I guess the deprecated code will be removed later. I don't think this is needed.